### ISSUE TITLE
Get consistency with category and tag functionality

### ISSUE THREAD

**Issue by micheal** [getnikola/nikola#2567]
Hi,

Newcomer to Nikola. One of the things I had a question on about Nikola is the difference between the tag and category functionality.

If you have tags, you can set a tag index page to be tag.html, with a directory `tags/` created for storing the tags. This `tags.html` index page functionality is different from the category functionality, which doesn't let you create a `categories.html` index page and instead creates a `categories/` directory and uses the `index.html` page in that directory as the index.

From the config:

~~~
# Final locations are:
# output / TRANSLATION[lang] / TAG_PATH / index.html (list of tags)
# output / TRANSLATION[lang] / TAG_PATH / tag.html (list of posts for a tag)
# output / TRANSLATION[lang] / TAG_PATH / tag.xml (RSS feed for a tag)
# (translatable)
TAG_PATH = "tags"

# See TAG_PATH's "list of tags" for the default setting value. Can be overwritten
# here any path relative to the output directory.
# (translatable)
TAGS_INDEX_PATH = "tags.html"
~~~

But for categories, that configuration is inconsistent:

~~~
# Final locations are:
# output / TRANSLATION[lang] / CATEGORY_PATH / index.html (list of categories)
# output / TRANSLATION[lang] / CATEGORY_PATH / CATEGORY_PREFIX category.html (list of posts for a category)
# output / TRANSLATION[lang] / CATEGORY_PATH / CATEGORY_PREFIX category.xml (RSS feed for a category)
# (translatable)
CATEGORY_PATH = "categories"
CATEGORY_PREFIX = ""
~~~

To sum up, the tags and categories functionality appears to be different from each other. I'd like for the categories functionality to behave like the tags functionality. That is, a `categories.html` page created alongside a `tags.html` page as the index page. I can help with creating a PR if there's interest.

**Comment by ralsina** [comment]
Categories are (very slowly) being phased out, because they are sort of an undefined thing. The new thing is sections. If you are starting a new site, I'd suggest going with sections instead.

Having said that, you are correct! This is asymmetrical and not pretty. A solution that doesn't break other people's sites by changing their URLs seems doable, so marking this as needing to be done, but not high priority.

Thanks for the bug report!

**Comment by felixfontein** [comment]
Did anyone already start implementing this? Otherwise I can simply add this while working on #2535.

**Comment by felixfontein** [comment]
(Does anyone also need a `TAG_PREFIX`? Or can I ignore that and just add `CATEGORIES_INDEX_PATH`?)

**Comment by micheal** [comment]
I haven't started yet. Things got a bit busy for me. I don't think I need a `TAG_PREFIX`. I set the category prefix to be empty anyway.
